static/bidder-info/resetdigital.yaml

```diff
@@ -0,0 +1,19 @@
+    endpoint: http://b-us-east14.resetdigital.co:9001
+    meta-info:
+      maintainer-email: biddersupport@resetdigital.co
+      app-media-types:
+        - banner
+        - video
+        - audio
+      site-media-types:
+        - banner
+        - video
+        - audio
+      supported-vendors:
+      vendor-id: 1162
+    usersync:
+      cookie-family-name: resetdigital
+      redirect:
+        url: https://sync.resetdigital.co/csync?pid=rubicon&redir={{redirect_url}}
+        support-cors: false
+        uid-macro: '$USER_ID'
```

Comment:
Recommended to follow the structure maintained by appnexus yaml
https://github.com/prebid/prebid-server/blob/06ac44bbca993745ffd4158c40f9042430fcbde2/static/bidder-info/appnexus.yaml#L1-L21